graphical + text update

s-a-kino opened the pull request:
- Graphics added to the login, home, login success, and logout pages
- Admin login page button text fixed
- Edited the wording of the text on the home page

s-a-kino committed:
graphical + text update

- Graphics added to the login, home, login success, and logout pages
- Admin login page button text fixed
- Edited the wording of the text on the home page

gracieorozco commented:
It looks great so far, but for the log in screens, I'm seeing a white bar below (Image 1). Is this how it looks like on your screen? Aside from that, I think putting a box around the content would help the user see the text better (Image 2). 

Image 1:
![image](https://user-images.githubusercontent.com/42789089/81026086-c9e32300-8e2d-11ea-9326-ab199982575f.png)

Image 2:
![image](https://user-images.githubusercontent.com/42789089/81025988-7ffa3d00-8e2d-11ea-8ddf-22c190e9134d.png)

s-a-kino committed:
added text boxes

Boxes placed around login, login success, logout, and admin login page text. Fixed image scaling to come soon.

s-a-kino committed:
updated login-related graphics

s-a-kino committed:
updated home page graphics

it MIGHT work

s-a-kino committed:
updated graphics

extended home graphics

HollyBakez reviewed (approved):
Looks good now, the hero banner scales nicely. Good Job.
Image:
![image](https://user-images.githubusercontent.com/41611254/81039289-74bf0580-8e5d-11ea-84f2-95c27e1bcbb1.png)
